src/app/game/game.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit } from '@angular/core';
+
+import { Answer } from '../models/answer';
+import { AnswerService } from '../services/answer.service';
+import { filter } from 'rxjs';
+
+@Component({
+  selector: 'app-game',
+  templateUrl: './game.component.html',
+  styleUrls: ['./game.component.sass']
+})
+export class GameComponent implements OnInit {
+  answer: Answer = { id: 0, word: '' };
```

Comment:
It's good practice, and something that is enforced in the Shure project to always include an accessibility modifier, `protected`, `private`, `public`

Reply:
This is helpful! It's also interesting that the test file isn't considered a subclass of GameComponent and therefore can't access the `protected` answer property.

Reply:
I think you'd have to do something like:

`class ChildClass implements GameComponent { ... }` 

to be able to properly access `protected` properties. The test file just creates in an instance of `GameComponent` therefore it can only access `public` members.

Something I recently discovered is that Angular templates _can_ access `protected` members but not `private` members. I haven't had a chance to use this information yet, but I've kept in my back pocket because it seems useful!